CLAUDE.md

```
@@ -1,12 +1,86 @@
+
+```bash
+pixi run fix-and-check-all   # Fix and check toolkit + Snakemake linting
+pixi run lint --check        # Validate Snakemake files with snakefmt
+pixi run download            # Run the Snakemake download workflow
+```
+
+## Architecture
+
+### Repository Layout
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)